hello.rb

```diff
@@ -292,6 +292,5 @@ def app_url
 # The scopes we are requesting (must match what we entered when
```

Comment:
🍹 Could we change the comment to reflect that the scopes can be modified in the app?